src/indexer-nibi/query/wasm.ts

```diff
@@ -0,0 +1,68 @@
+import { defaultUserContract } from "../defaultObjects"
+import { convertObjectToPropertiesString, doGqlQuery, gqlQuery } from "../gql"
+import {
+  GQLQuery,
+  GQLUserContract,
+  GQLWasmGqlUserContractsArgs,
+} from "../gql/generated"
+
+export type QueryWasmArgs = {
+  userContracts?: GQLWasmGqlUserContractsArgs
+}
+
+export interface GqlOutWasm {
+  wasm?: GQLQuery["wasm"]
+}
+
+export type GqlWasmFields = Partial<{
+  userContracts?: Partial<GQLUserContract>
+}>
+
+export const wasmQueryString = (
```

Comment:
This code should likely be more like src\indexer-nibi\query\communityPool.ts

This pattern is reserved for multi-level deep modules like oracle, perps, and stats

Reply:
This alternative is a more straightforward approach

Reply:
wasm is the same deep module as stats, oracle, etc.